### PR TITLE
142689 - BESS Builder Service WebJob Logging and Configuration

### PR DESCRIPTION
**Pull request by rupali15127**
This PR consists code changes of [PBI 142689](https://dev.azure.com/ukhydro/Exchange%20Set%20Service/_workitems/edit/142689)

Kindly review the same.

### CODE REVIEW

File: UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Program.cs [hunk 1]
```diff
@@ -1,21 +1,144 @@
...
+using UKHO.PeriodicOutputService.Common.Configuration;
 
 namespace UKHO.BESS.BuilderService
 {
     [ExcludeFromCodeCoverage]
     public static class Program
     {
-        static void Main()
+        private static readonly InMemoryChannel s_aIChannel = new();
+        private static readonly string s_assemblyVersion = Assembly.GetExecutingAssembly().GetCustomAttributes<AssemblyFileVersionAttribute>().Single().Version;
```
Comment (harshal11869) [inline UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Program.cs:22]:
```suggestion
        private static readonly InMemoryChannel aiChannel = new();
        private static readonly string assemblyVersion = Assembly.GetExecutingAssembly().GetCustomAttributes<AssemblyFileVersionAttribute>().Single().Version;
```

File: UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/BessBuilderServiceJob.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.Logging;
+using UKHO.PeriodicOutputService.Common.Helpers;
+using UKHO.PeriodicOutputService.Common.Logging;
+
+namespace UKHO.BESS.BuilderService
+{
+    public class BessBuilderServiceJob
+    {
+        private readonly ILogger<BessBuilderServiceJob> logger;
```
Comment (harshal11869) [inline UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/BessBuilderServiceJob.cs:9]: Add one blank after line no. 9

**Review by harshal11869 — CHANGES_REQUESTED** [review]

File: UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/BessBuilderServiceJob.cs (hunk 2 above)
Comment (rupali15127) [reply UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/BessBuilderServiceJob.cs:9]: Resolved.

File: UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Program.cs (hunk 1 above)
Comment (rupali15127) [reply UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Program.cs]: Resolved

**Review by harshal11869 — APPROVED** [review]
LGTM.

**Review by Nadeem14456 — APPROVED** [review]
LGTM.

**Review by Mayuresh-Gawde — APPROVED** [review]
LGTM from QA!!

**Pull request merged**